.storybook/main.js

```diff
@@ -30,18 +29,19 @@ module.exports = {
     checkOptions: {},
     reactDocgen: 'react-docgen-typescript',
     reactDocgenTypescriptOptions: {
-      shouldExtractLiteralValuesFromEnum: true,
-      shouldRemoveUndefinedFromOptional: true,
+      compilerOptions: {
+        allowSyntheticDefaultImports: false,
+      },
       propFilter: prop => {
-        // Remove styled props that do not have a description.
-        if (!styledProps.has(prop.name) && !prop.description) return;
-
-        if (config.reactDocgenTypescriptOptions.propFilter) {
-          return config.reactDocgenTypescriptOptions.propFilter(prop);
+        if (['theme'].includes(prop.name)) {
+          return false;
         }
 
-        return true;
+        return prop.parent
```

Comment:
Any component that extended the props of another component was not rendering the extended props within its props table. Updated this logic to render those props.